packages/backend/src/peripherals/database/AssetRepository.ts

```diff
@@ -70,8 +72,8 @@ export class AssetRepository extends BaseRepository {
     return row ? toAssetDetailsRecord(row) : undefined
   }
 
-  async getDetailsByAssetHashes(
-    assetHashes: AssetHash[]
+  async getDetailsByAssetHashesOrTypeHashes(
```

Comment:
Selecting by `asset_type_hash` is not necessary. `asset_details` table has ERC20 assets added with 'asset_hash' set to their type hashes. This function was fine before.